CMakeLists.txt

```diff
@@ -190,15 +221,15 @@ set_property(TARGET viam_rust_utils PROPERTY IMPORTED_LOCATION ${viam_rust_utils
 
 install(
   IMPORTED_RUNTIME_ARTIFACTS viam_rust_utils
-  LIBRARY COMPONENT runtime
+  LIBRARY COMPONENT viam-cpp-sdk_runtime
```

Comment:
Made this change here and elsewhere based on some suggestions I saw online to namespace the components.